service/worker/scanner/scanner.go

```diff
@@ -117,18 +119,16 @@ func New(
 
 // Start starts the scanner
 func (s *Scanner) Start() error {
-	backgroundActivityContext := context.WithValue(context.Background(), scannerContextKey, s.context)
-	if s.context.cfg.ExecutionScannerConfig.Enabled() {
-		backgroundActivityContext = context.WithValue(backgroundActivityContext, executions.ScannerContextKey, executions.ScannerContext{
-			Resource:                     s.context.Resource,
-			Scope:                        s.context.Resource.GetMetricsClient().Scope(metrics.ExecutionsScannerScope),
-			ScannerWorkflowDynamicConfig: s.context.cfg.ExecutionScannerConfig,
-		})
-	}
-	backgroundActivityContext = context.WithValue(backgroundActivityContext, executions.FixerContextKey, executions.FixerContext{
-		Resource: s.context.Resource,
-		Scope:    s.context.Resource.GetMetricsClient().Scope(metrics.ExecutionsFixerScope),
-	})
+	backgroundActivityContext, taskListNames :=
```

Comment:
why taskListNames is needed to be returned from startExecutionWorkflowWithRetry?

Reply:
We dont know those tasklists before call.

Reply:
I think we need the names to create workers to poll from those tasklists.